tests/integration/apigateway/test_apigateway_common.snapshot.json

```diff
@@ -20,7 +20,7 @@
         "httpMethod": "POST",
         "methodIntegration": {
           "cacheKeyParameters": [],
-          "cacheNamespace": "5l4d79",
```

Comment:
nit: we could add a transformer (e.g., `regex(..)` to store a placeholder for this value (to make the test repeatable against AWS (without updating the snapshot) as well). 👍

Reply:
Right, yes! Good catch. It is repeatable at the moment because we're skipping the verification of the snapshot because methodIntegration format is very not in parity, but I'll add the transformer right now!